### DISCUSSION TITLE
Display label value in Stat Chart

### DISCUSSION THREAD

**Discussion by ibakshay** [perses/perses#2393] [Q&A]
Hello Perses Experts,

Could you please help me display the "version" label in a Stat Chart using Perses?
I'm currently using this PromQL query:
```
count by (version)(prometheus_build_info{job="$prometheus"})
```
The current output shows:
```
{version="2.54.1"}     1
```
Instead of displaying "1" in the Stat Chart, I would like it to show "2.54.1". How can I achieve this?

### Screenshot 1

<img width="385" alt="image" src="https://github.com/user-attachments/assets/0e7b366a-ea1e-44fe-8992-7c706468f31f">

### Screenshot 2

<img width="1065" alt="image" src="https://github.com/user-attachments/assets/6dd59fe1-cc84-40c2-8800-4ae72013db7e">

**Comment by AntoineThebaud** [comment]
Hello, please post in the existing issue for that topic instead https://github.com/perses/perses/issues/2340
